CMakeLists.txt

```diff
@@ -244,12 +244,13 @@ endfunction()
 read_repo_version(llvmproject llvm-project)
 read_repo_version(picolibc picolibc)
 
+file(GLOB llvm_project_patches ${CMAKE_CURRENT_SOURCE_DIR}/patches/llvm-project/*.patch)
```

Comment:
According to CMake docs GLOB is not recommended. Please make sure it works in every use case.

Reply:
I replaced the `GLOB` with explicit list.